src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -253,6 +263,16 @@ public class CrosscheckFingerprints extends CommandLineProgram {
             doc = "One or more input files (or lists of files) with which to compare fingerprints.", minElements = 1)
     public List<String> INPUT;
 
+    @Argument(doc = "A tsv with two columns and no header which maps the input files to corresponding indices; to be used when index " +
+            "files are not located next to input files. First column must match the list of inputs. ", optional = true)
+    public File INPUT_INDEX_MAP;
```

Comment:
It looks like changing these away from `File` causes a number of other problems, especially `PicardHtsPath`. But even `Path` causes a few downstream changes that would probably be fixed by writing `mypath.toFile()` in a bunch of places. If you think this is a worthwhile pursuit, I can try to hunt down the appropriate changes for this. But it might be best saved for a full rewrite where all of the inputs are appropriately updated.

Reply:
If it's a pain we can put it off.  Thanks for taking a look.